src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/eval/validate.test.ts

```diff
@@ -352,23 +352,7 @@ describe('EVAL Validation', () => {
   });
 
   test('date math', () => {
-    evalExpectErrors('from a_index | eval 1 anno', [
-      'EVAL does not support [date_period] in expression [1 anno]',
-    ]);
-    evalExpectErrors('from a_index | eval col0 = 1 anno', [
-      "Unexpected time interval qualifier: 'anno'",
-    ]);
-    evalExpectErrors('from a_index | eval now() + 1 anno', [
-      "Unexpected time interval qualifier: 'anno'",
-    ]);
```

Comment:
Same here. We no longer check that the qualifier is valid. Happy to add this back if desired.